src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -110,10 +115,21 @@ <h3>Launch with</h3>
               ><img src="../assets/images/thirdparty/cavatica.png" /> Cavatica
             </a>
           </div>
+
+          <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'gxformat2'">
+            <app-multi-cloud-launch
+              [cloudInstances]="cloudInstances"
+              [usersCloudInstances]="usersCloudInstances"
+              [languagePartner]="partner.GALAXY"
+              [launchUrlAppend]="'%23workflow/' + workflow?.full_workflow_path"
```

Comment:
this looks like the TRS ID. calling it TRS ID may be more suitable

Reply:
I figured it wouldn't always look like the TRS id so I gave it a more generic name. Renamed it a bit though